### PR TITLE
I have done important changes to the files. 

### PR DESCRIPTION
**Pull request by RATFIVE**

### CODE REVIEW

File: add.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Example
+
+When working on a software project, you might make changes to several files to add a new feature. Once you've tested your changes and are satisfied with them, you can commit those changes. This creates a new commit in your Git history that represents the state of your project at that point in time.
+
+## Code Snippet
+
+Here are some basic Git commands related to commits:
+
+- `git commit -m "Your descriptive message here"`: This command creates a new commit with the changes you've staged using `git add`, and includes a descriptive message of your choice.
```
Comment (Tomson909) [inline add.md:15]: MAybe more examples. A link to the official documentation would be cool.

**Review by Tomson909 — COMMENTED** [review]
Good work but some features could be added

File: add.md (hunk 1 above)
Comment (RATFIVE) [reply add.md:15]: I am not sure. I think it is better to ad the pandas github fork to that ssh key.

Comment (Tomson909) [reply add.md:15]: Ok youre right. But next time you will be fired.

Comment (RATFIVE) [reply add.md:15]: Please rename the file to commit, I am bussy today

Comment (RATFIVE) [reply add.md:15]: Are you able to do that or not?

Comment (Tomson909) [reply add.md:15]: I cant do that. I think i dont have the rights to do that.

Comment (RATFIVE) [reply add.md:15]: Ok I will try it

Comment (Tomson909) [reply add.md]: ok nice you changed the name. I will merge. Please check my code.

Comment (RATFIVE) [reply add.md]: Heute Tischtennis?

Comment (Tomson909) [reply add.md]: Wann haszt du heute zeit? Ich hab bock heute abend.

Comment (Tomson909) [reply add.md]: Yalla ich hab bock.

Comment (RATFIVE) [reply add.md]: 18:30 ins Fitz

**Pull request merged**